incubator/java-openliberty/image/project/pom.xml

```diff
@@ -155,7 +155,71 @@
+                                <runtimeArtifact>
+                                    <groupId>${liberty.groupId}</groupId>
+                                    <artifactId>${liberty.artifactId}</artifactId>
+                                    <version>${version.openliberty-runtime}</version>
+                                    <type>zip</type>
+                                </runtimeArtifact>
+                            </configuration>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+            </build>
+        </profile>
+
+
+        <!-- This profile is used during "appsody build" where just any image will do, so it's
+            better to start small, and there's no benefit in caching a more complete artifact.  -->
+        <profile>
+            <id>appsody-build</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>io.openliberty.tools</groupId>
+                            <artifactId>liberty-maven-plugin</artifactId>
+                            <version>${version.liberty-maven-plugin}</version>
+                            <configuration>
+                                <runtimeArtifact>
+                                    <groupId>${liberty.groupId}</groupId>
+                                    <artifactId>openliberty-kernel</artifactId>
+                                    <version>${version.openliberty-runtime}</version>
+                                    <type>zip</type>
+                                </runtimeArtifact>
+                                <installDirectory>${project.build.directory}/liberty/wlp</installDirectory>
+                            </configuration>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+            </build>
+        </profile>
+
+        <!-- Last, if no other profile is activated, we are in host mode, where we don't assumed an already-installed OL -->
+        <profile>
+            <id>default-profile</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>io.openliberty.tools</groupId>
+                            <artifactId>liberty-maven-plugin</artifactId>
+                            <version>${version.liberty-maven-plugin}</version>
+                            <configuration>
+                                <runtimeArtifact>
+                                    <groupId>${liberty.groupId}</groupId>
+                                    <artifactId>openliberty-kernel</artifactId>
```

Comment:
Is there a reason why we are using the kernel runtime for hostmode/build and the runtime for stack package / dev?

Reply:
It's smaller, more fine-grained.  We use the bigger one in stack package/dev since we can bake it into the stack image.

Though in `appsody build`, it'd also typically be layered.. it still is a big difference to download initially (150 vs 50 MB, IIRC), and it seems now and then we've hung up Maven with the bigger zip.